VideoWeb/VideoWeb/ClientApp/src/app/shared/select-media-devices/select-media-devices.component.ts

```diff
@@ -34,17 +34,34 @@ export class SelectMediaDevicesComponent implements OnInit, OnDestroy {
 
     constructor(
         private userMediaService: UserMediaService,
-        private mediaStreamService: MediaStreamService,
+        private userMediaStreamSerivce: UserMediaStreamService,
         private logger: Logger,
         private translateService: TranslateService,
         private profileService: ProfileService,
         private videoFilterService: VideoFilterService
     ) {}
 
     ngOnInit() {
-        this.userMediaService.connectedDevices$.pipe(takeUntil(this.destroyedSubject)).subscribe(connectedDevices => {
-            this.availableCameraDevices = connectedDevices.filter(device => device.kind === 'videoinput');
-            this.availableMicrophoneDevices = connectedDevices.filter(device => device.kind === 'audioinput');
+        this.userMediaService.connectedVideoDevices$.pipe(takeUntil(this.destroyedSubject)).subscribe(cameraDevices => {
+            this.availableCameraDevices = null;
```

Comment:
why do you set to null and then the new incoming variable?

Reply:
Left over code from something else. Removed it.